src/physics/src/elastic_cable.C

```diff
@@ -358,6 +377,31 @@ namespace GRINS
               }
             return;
           }
+
+        if( is_stress )
+          {
+            if( _stress_indices[0] == quantity_index )
+              {
+                // Need to convert to Cauchy stress
+                value = tau(0,0)/std::sqrt(I3);
+              }
+            else
+              {
+                libmesh_error();
+              }
+          }
+        if( is_force )
+          {
+
+            if( _force_indices[0] == quantity_index )
+              {
+                value = tau(0,0)/std::sqrt(I3)*_A;
```

Comment:
We need to add a comment that this is force in the deformed configuration. This won't make a difference in the small strain case (or much of one anyway), but for large strains, this would be important.